frontend/src/SettingsView.svelte

```diff
@@ -257,13 +239,13 @@
       {/each}
     </div>
     <div style:margin="1em 0 2em 0">
```

Comment:
For some reason this change seems to have affect the arrangement of buttons in the settings pane.
- Before: 
<img width="539" alt="image" src="https://github.com/redballoonsecurity/ofrak/assets/99368685/dc6dbb37-a6bc-4453-ae1e-4b901a9742c0">

- After: 
<img width="236" alt="image" src="https://github.com/redballoonsecurity/ofrak/assets/99368685/b2070846-ab39-4058-9dd1-5105987286d2">

Reply:
This is very weird, i can not figure out how it was arranged like that previously without an explicit class associated, parent blocks are flex column... will just make a new class